src/settings/ShowCapabilities.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import { stripesConnect, useOkapiKy } from '@folio/stripes/core';
+import { Button, Pane, Row, Col, SearchField } from '@folio/stripes/components';
+
+const ShowCapabilities = () => {
+  const ky = useOkapiKy();
+  const [capabilitiesResults, setCapabilitiesResults] = useState({});
+  const [capabilitySetsResults, setCapabilitySetsResults] = useState({});
+  const [query, setQuery] = useState('');
+
+  const searchParams = {
+    limit: 1000,
```

Comment:
Can you pull a limit from stripes-config instead -- don't we have a constant there, or in stripes-core somewhere? Magic numbers make me twitchy.